rose/client/game.py

```diff
@@ -60,18 +62,18 @@ def car(self):
     # Handling client events
 
     def client_connected(self):
-        print 'client connected: joining as', self.name
+        log.info('client connected: joining as', self.name)
```

Comment:
When converting prints to logging, we need format string and args - for example:

    log.info('client connected: %s', self.name)